lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -933,7 +933,7 @@ function dosomething_campaign_get_goals_times($node) {
   $timing = [
     "today" => new DateTime(),
     "start_date" => new DateTime($high_season[0]['value']),
-    "end_date" => new DateTime($high_season[0]['value2']),
+    "end_date" => date_modify(new DateTime($high_season[0]['value2']), '+23 hours +59 minutes +59 seconds'),
```

Comment:
This is used by the win module right below it, and it hasn't really had any problems.